src/dag-node/addLink.js

```diff
@@ -5,7 +5,7 @@ const cloneLinks = dagNodeUtil.cloneLinks
 const cloneData = dagNodeUtil.cloneData
 const toDAGLink = dagNodeUtil.toDAGLink
 const DAGLink = require('../dag-link')
-const DAGNode = require('./index')
+const DAGNode = require('./dagNode.js')
```

Comment:
We normally don't require with the `.js` extension. I know that in this code base we are not consistent, but it would be great to do this for new changes.

So please change this, if you're making changes anyway. If you don't, nevermind it's not a blocker for getting merged.